### PR TITLE
Added emitting `import` directive for white-listed libraries

### LINKED ISSUE #1: Imports from node_modules are not included
Consider the following code:

```ts
import {OnInit} from "@angular/core";
class MyComponent implements OnInit {
    ngOnInit() {};
}
```

### Expected output:
```ts
import {OnInit} from "@angular/core";
declare class MyComponent implements OnInit {
    ngOnInit(): void;
}
```

### Actual output:
```ts
declare class MyComponent implements OnInit {
    ngOnInit(): void;
}
```
This results in an error when importing this generated declaration file, because `OnInit` cannot be found.

### PR DESCRIPTION
**Pull request by timocov**
Fixes #1 

According https://github.com/timocov/dts-bundle-generator/issues/1#issuecomment-334972526 this PR uses second approach (import type from `main` file of the library) to solve the problem.

### CODE REVIEW

File: src/bundle-generator.ts [hunk 1]
```diff
@@ -155,3 +198,14 @@ function spacesToTabs(text: string): string {
 		return '\t'.repeat(substring.length / 4);
 	});
 }
+
+function getLibraryName(libraries: string[], fileName: string): string | null {
+	return libraries.find((library: string) => {
+		return fileName.indexOf(`node_modules/${library}`) !== -1;
```
Comment (timocov) [inline src/bundle-generator.ts]: Possibly need to check that `fileName` ends with `node_modules/${library}` or after `node_modules/${library}` in `fileName` is `/`

**Comment by ajafff** [comment]
Looks good, but I have one problem with this solution:
First of all I'm lazy and don't want to list all used dependencies. I may even miss one.
And I don't want to update my project's build every time I add another dependency.

Is it possible to add a wildcard to simply add imports for all external libraries used?

**Comment by timocov** [comment]
I have done it like this because in our case we export public API from internal files and do not want to export more than needed and want to control what we export exactly :)

I think yes, it is good idea to have such wildcard - I have created another issue for it #6.
With libraries such angular/rxjs/any other library with types inside package seems there is no problem to do it, but with libraries with typings from `@types` may be some problems - I need to check it.

**Comment by timocov** [comment]
The problem may be because some types is "non-imported" (for example old versions of `jquery`) and should be added to files with triple-slash directive (see #3).

I am not sure if "non-`@types`" library can have such problems too.

**Pull request merged**